tests/functional/storageclass/test_ec_default_pool_and_sc.py

```
@@ -0,0 +1,262 @@
+        rbd_sc_name = constants.DEFAULT_STORAGECLASS_RBD
+
+        # Step 1: Enumerate all PVCs in openshift-storage and verify all are Bound
+        all_pvc_objs = pvc.get_all_pvc_objs(namespace=namespace)
+        assert all_pvc_objs, "No PVCs found in openshift-storage namespace"
+        log.info("=== PVC inventory in openshift-storage ===")
+        not_bound = []
+        for pvc_obj in all_pvc_objs:
+            status = pvc_obj.get().get("status", {}).get("phase", "Unknown")
+            log.info(
+                f"  PVC: {pvc_obj.name}, StorageClass: {pvc_obj.backed_sc}, "
+                f"Status: {status}"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)